src/Signers/MultiSig.js

```diff
@@ -49,8 +49,8 @@ class MultiSig {
      */
     async getPubK() {
         try {
-            if (!this.publicKeys || this.publicKeys.length == 0) {
-                throw "Need public keys"
+            if (!this.publicKeys || +this.publicKeys.length === 0) {
```

Comment:
I would suggest using Number() or parseInt() here for consistency and easy to read instead of `+`.